docs/source/publish-docs.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish Documentation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    if: github.repository_owner == 'bluesky'
```

Comment:
```suggestion
    # This conditional ensures that forks on GitHub do not attempt to publish
    # documentation, only the "upstream" one. (If fork _did_ try to publish they
    # would fail later below because they would not have access to the secret
    # credentials. But it's better to opt out here.)
    if: github.repository_owner == 'YOUR_GITHUB_USERNAME_OR_ORGANIZATION'
```